src/aliases/api/types/alias.go

```diff
@@ -0,0 +1,61 @@
+package types
+
+import aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
+
+type Alias struct {
```

Comment:
Following `/nodes` convention we should use naming such as `AliasRequest` and `AliasResponse` in addition we would have `formatters` to parse Request/Response from/to Entities